cosmos/operators/docker.py

```diff
@@ -42,8 +43,10 @@ class DbtDockerBaseOperator(AbstractDbtBaseOperator, DockerOperator):  # type: i
     def __init__(
         self,
         image: str,  # Make image a required argument since it's required by DockerOperator
+        profile_config: ProfileConfig | None = None,
```

Comment:
Does this profile work with [ProfileMapping](https://astronomer.github.io/astronomer-cosmos/profiles/index.html#using-a-profile-mapping) or only with a given [profiles.yml](https://astronomer.github.io/astronomer-cosmos/profiles/index.html#using-your-own-profiles-yml-file) file?

Reply:
I'm not familiar with ProfileMapping, so I'm not sure. I only tested this with `profiles.yml`.